CONTRIBUTING.md

```diff
@@ -1,19 +1,24 @@
-# Contribute to the project
+# Contribute to Strapi
 
-Strapi is an open-source project administered by [the Strapi team](http://strapi.io).
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
```

Comment:
First off all

Reply:
Well, "first off" works or "first of all".

First off all doesn't @soupette :)